test.properties

```diff
@@ -11050,15 +11050,15 @@
 
     workspaces.client.extension.names[liferay-sample-workspace]=\
         liferay-sample-batch,\
-        liferay-sample-custom-element-1,\
-        liferay-sample-custom-element-2,\
-        liferay-sample-custom-element-3,\
+        liferay-sample-custom-element-1-vanilla-js,\
+        liferay-sample-custom-element-2-create-react-app,\
```

Comment:
we have to be careful here.  The upper limit for LXC is 63 characters for 
`${serviceId}-${projectId}-${env}`

For instance, if I try to deploy this sample to LXC Staging cluster, it will fail because the final domain will be

https://liferaysamplecustomelement2createreactapp-exti4f5cloudstaging-extuat.lfr.st

This is too big, the first part of the domain is 69 characters.  DNS record limit is 63 at the most "per subdomain".

So we need to try to reduce the size of these project names, unless we have some type of "reduced" LCP.json projectId.

Reply:
Let me also propose showing a very worrying error message to the user when he surpasses that limit (when building or creating the project), instead of making them and us spend several weeks trying to find out what's going on :sweat_smile: